tests/v3_api/test_secrets.py

```diff
@@ -191,11 +191,11 @@ def fin():
 
 
 def validate_workload_with_secret(p_client, workload,
-                                  type, ns_name, keyvaluepair,
+                                  type, ns_name, keyvaluepair, podcount=1,
```

Comment:
@soumyalj Can you include the podcount variable as the last argument?

Reply:
Yes, made it the last argument